src/content/docs/en/guides/backend/xata.mdx

```diff
@@ -6,9 +6,142 @@ service: Xata
 stub: true
 ---
 import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+import FileTree from '~/components/FileTree.astro'
 
-[Xata](https://xata.io) is a **Serverless Data Platform** that combines the features of a relational database, a search engine, and an analytics engine by exposing a single consistent REST API.
+
+[Xata](https://xata.io) is a **Serverless Data Platform** because it combines the features of a relational database, a search engine, and an analytics engine by exposing a single consistent REST API.
+
+## Adding a database with Xata
+
+### Prerequisites
+
+- A [Xata](https://app.xata.io/signin) account with a created database (you can use the sample database from the Web UI).
+- An Access Token (`XATA_TOKEN_API`).
+
+### Environment configuration
```

Comment:
So, I just want to be clear here, because we normally ONLY include the stuff required to make 3rd parties work with Astro: Is this environment config an *Astro-specific* configuration step, or is this something that Xata docs describe how to do to have Xata working with any project?

My gut feeling is that this is not something for this guide, but something that could be added as a pre-requisite line (i.e. Xata CLI configured globally, maybe with a link to Xata docs for this?)

These guides can get pretty long, and we try to include as little as possible that's (a) documented elsewhere and (b) subject to change in someone else's project that we wouldn't necessarily know about.

Reply:
Yes. It's not Xata specific. I was trying to follow the lead from the Firebase recipe. I'll add a note to the prerequisites about having the project linked to a DB and pointing to our docs. 


But I'll leave here the `env.d.ts` part because that's Astro specific and I think it could help, wdyt??

Reply:
Yes, that was my initial thinking! It is Astro specific. Good instinct!